src/decorators.ts

```diff
@@ -144,6 +144,14 @@ export function Acl(...acl: string[]) {
     const parameterNames = functionArguments(execute)
     const types = Reflect.getMetadata('design:paramtypes', UserCommand, key)
 
+    if (!types) {
```

Comment:
Is it possible that it would return null for other reasons? Are we sure null is returned only when not set?